sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUdfs.scala

```diff
@@ -81,6 +82,8 @@ private[hive] case class HiveSimpleUdf(funcWrapper: HiveFunctionWrapper, childre
 
   override def nullable: Boolean = true
 
+  override def checkInputDataTypes(): TypeCheckResult = TypeCheckResult.TypeCheckSuccess
```

Comment:
So, we do not check input data types for HiveSimpleUdf? Is it because we cannot get the required data types from a Hive function?